examples/dmd/de_dg_advection_greedy.cpp

```diff
@@ -0,0 +1,1625 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2023, Lawrence Livermore National Security, LLC
```

Comment:
PR 262 updates the year 2023 to 2024, but it doesn't include this file. Please update here.